zerver/webhooks/gogs/tests.py

```diff
@@ -9,6 +11,14 @@ class GogsHookTests(WebhookTestCase):
     URL_TEMPLATE = "/api/v1/external/gogs?&api_key={api_key}"
     FIXTURE_DIR_NAME = 'gogs'
 
+    def build_webhook_url(self, branches=None):
```

Comment:
Hmm.. sorry, but I am not sure what you mean, are you suggesting that instead of passing `branches`, we should pass in `**kwargs`? Or are you suggesting that we should modify the original definition in `WebhookTestCase` and pass in `**kwargs` there?